etc/kayobe/ansible/roles/nova-compute-drain/tasks/setup.yml

```diff
@@ -0,0 +1,29 @@
+---
+
+- name: Initiate openstack cli virtualenv
+  pip:
+    virtualenv: "{{ nova_compute_drain_venv }}"
+    name:
+      - pip
+      - setuptools
+    state: latest
+    virtualenv_command: /usr/bin/python3.6 -m venv
+  run_once: true
+  delegate_to: "{{ nova_compute_drain_delegate_host }}"
+  vars:
+    # NOTE: Without this, the delegate ansible_host variable will not
+    # be respected when using delegate_to.
+    ansible_host: "{{ hostvars[nova_compute_drain_delegate_host].ansible_host | default(nova_compute_drain_delegate_host) }}"
+
+- name: Install openstack CLI tools in virtualenv
+  pip:
+    virtualenv: "{{ nova_compute_drain_venv }}"
+    name:
+      - python-openstackclient
```

Comment:
Should we pin it, or use upper constraints? (as was done before)

Reply:
Good point, In train where I was testing this we couldn't use: ` --os-compute-api-version 2.25`. So I needed to install a later version IIRC. I'll re-add constraints.